src/components/CaptureMatching/CaptureImage.js

```diff
@@ -10,6 +10,40 @@ import SkipNextIcon from '@material-ui/icons/SkipNext';
 import { makeStyles } from '@material-ui/core/styles';
 import { getDateTimeStringLocale } from 'common/locale';
 
+function Country({ lat, lon }) {
+  const [content, setContent] = React.useState('');
+  if (lat === 'undefined' || lon === 'undefined') {
+    setContent('No data');
+  }
+
+  React.useEffect(() => {
+    setContent('loading...');
+    fetch(
+      `${process.env.REACT_APP_QUERY_API_ROOT}/countries?lat=${lat}&lon=${lon}`
+    )
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        } else if (res.status === 404) {
+          setContent('Can not find country at this place');
```

Comment:
Maybe we just fall back to displaying raw lat, lon in this case, or nothing at all.
I don't think this message is very useful to the user.

Reply:
But if we can't find country for the coordinates, usually it means something wrong, so we'd better let people know that, and if we just simple show lat/lon here, it's a bit confusion for people, why sometimes it's country name and sometimes it's not. I can make this message more clear: `cannot find country at [lat, lon]`

Reply:
That works for me